src/Microsoft.Azure.WebJobs.Host/Bindings/ExecutionContext/ExecutionContext.cs

```diff
@@ -32,5 +32,10 @@ public class ExecutionContext
         /// A host can set this via <see cref="CoreJobHostConfigurationExtensions.UseCore(JobHostConfiguration, string)"/>
         /// </remarks>
         public string FunctionAppDirectory { get; set; }
+
+        /// <summary>
+        /// Gets or sets value for retry count.
+        /// </summary>
```

Comment:
can we expose https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/RetryContext.cs instead of just `RetryCount` ? This will ensure out-of-proc language functions have access to retry count as well as the exception details for the failure

Reply:
Good idea. Fixed.